build/test.sh

```diff
@@ -63,9 +64,18 @@ check_dependencies() {
         echo "Please install MinIO using 'make install-minio' and try again."
         exit 1
     fi
+
+    # A test (CRDSuite) that runs as part of `make test` requires at least one CRD to
+    # be present on the cluster. That's why we are checking that `csi-hostpath-driver`
+    # installed before running tests.
+    if ! ${SCRIPT_DIR}/local_kubernetes.sh check_csi_hostpath_driver_installed ; then
+        echo "Please install CSI hostpath driver using 'make install-csi-hostpath-driver' and try again."
+        exit 1
+    fi
```

Comment:
Should is say that CSI hostpath driver is not installed or it should say CRDs are not installed in the cluster, but a test requires at least one CRD to be available.

Reply:
something like

```
CRDs are not installed on the cluster but a test (CRDSuite) requires at least one CRD to be available on the cluster. One can be installed by running make install-csi-hostpath-driver command.
```

Reply:
fixed